src/kaleidoscope/plugin/IdleLEDs.cpp

```diff
@@ -56,7 +59,47 @@ EventHandlerResult IdleLEDs::onKeyswitchEvent(Key &mapped_key, KeyAddr key_addr,
   return EventHandlerResult::OK;
 }
 
+uint16_t PersistentIdleLEDs::settings_base_;
+
+EventHandlerResult PersistentIdleLEDs::onSetup() {
+  settings_base_ = ::EEPROMSettings.requestSlice(sizeof(uint16_t));
+
+  // If idleTime is max, assume that EEPROM is uninitialized, and store the
+  // defaults.
+  uint16_t idle_time;
+  KeyboardHardware.storage().get(settings_base_, idle_time);
+  if (idle_time == 0xffff) {
+    KeyboardHardware.storage().put(settings_base_, idle_time);
```

Comment:
Isn't this supposed to read
```cpp
KeyboardHardware.storage().put(settings_base_, idle_time_limit);
```
Otherwise it will store `0xffff` instead of the defaults (as the comment suggests).

Reply:
Good catch, thank you!